internal/resources/ibm/resource_instance.go

```diff
@@ -133,6 +141,13 @@ var ResourceInstanceUsageSchema = []*schema.UsageItem{
 	{Key: "sysdig-secure_VM_NODE_HOUR", DefaultValue: 0, ValueType: schema.Float64},
 	{Key: "aiopenscale_RESOURCE_UNITS", DefaultValue: 1, ValueType: schema.Float64},
 	{Key: "aiopenscale_MODELS_PER_MONTH", DefaultValue: 1, ValueType: schema.Float64},
+	{Key: "dns-svcs_ITEMS", DefaultValue: 1, ValueType: schema.Int64},
+	{Key: "dns-svcs_MILLION_ITEMS", DefaultValue: 1, ValueType: schema.Int64},
+	{Key: "dns-svcs_MILLION_ITEMS_CREXTERNALQUERIES", DefaultValue: 1, ValueType: schema.Int64},
+	{Key: "dns-svcs_NUMBERGLB", DefaultValue: 1, ValueType: schema.Int64},
```

Comment:
The pricing plan indicates $0.03681117 USD per GLB Instances per hour for this metric. I think this means that for 1 GLB, the price would be 0.0368 x the number of hours used in a month. There will need to be another usage variable to accept the number of hours the GLB is running in a month in order to fully calculate this cost component.  The unit in this case would be Hours instead of NUMBERGLB

Reply:
Thanks! I added a new commit. Is that what you meant?

Reply:
@hiltol Thanks! I pushed a change to introduce those usage variables to account for the hours. Let me know if it addresses it